helix-rest/src/main/java/org/apache/helix/rest/metadatastore/RoutingData.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.helix.rest.metadatastore;
```

Comment:
This is an interface class; therefore, we want to keep it generic. This means that this interface class should be independent of ZK.
Also please refrain from using internal details in comments. This is an open source project.